ci-scripts/unit-tests.sh

```diff
@@ -1,130 +1,125 @@
 #!/usr/bin/env bash
-set -e
 set -o pipefail
 
+# -azure argument used by azure hyperspace pipeline
+if [[ $1 == '-azure' ]]; then
```

Comment:
let's call it --azure

I would associate single dashes with 'aliased' names, so like -a

Basically, just change to --azure or support --azure OR -a

Reply:
nvm it was already changed lol

I look through commits from top down when new things come